test/libsolidity/semanticTests/cleanup/cleanup_bytes_types_shortening.sol

```diff
@@ -12,6 +12,6 @@ contract C {
 }
 // ====
 // compileToEwasm: also
-// compileViaYul: also
+// compileViaYul: true
```

Comment:
You can revert this change now and the test will still pass, won't it?

Reply:
Nope, the test does not pass if I revert this change.
New Code and OId Code Generators give conflicting outputs

Reply:
Ah, right, now I see the diff in the expectation correctly.
Then please split this test into two tests, one for old and one for new codegen and adjust the comment in the test case accordingly.